src/WebJobs.Script/Config/ScriptSettingsManager.cs

```diff
@@ -68,6 +68,31 @@ public virtual string AzureWebsiteUniqueSlotName
             }
         }
 
+        public virtual string ApplicationInsightsInstrumentationKey
+        {
```

Comment:
I wanted to be able to override this in app.config, then fall back to Environment variables. It seems like that would be generally useful so maybe we can extend this to other properties?

In this case, I have a machine env var with my app insights key. I don't want all tests logging to app insights -- so I set it to null in the app.config, then only turn it on for the tests that need it.